src/python/grpcio/grpc/_server.py

```diff
@@ -959,6 +958,17 @@ def add_secure_port(self, address, server_credentials):
     def start(self):
         _start(self._state)
 
+    def wait_for_termination(self):
+        termination_event = threading.Event()
+
+        with self._state.lock:
+            if self._state.stage is _ServerStage.STOPPED:
+                raise ValueError('Failed to wait for a stopped server.')
```

Comment:
Why raise an exception at all. If we are already stopped it should just return.

Reply:
Make sense. And it still align with the semantic of blocking until the server is stopped.